1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts

```diff
@@ -1,3 +1,11 @@
 export function reverseAString(str: string) {
-    return str.split("").reverse().join("");
+    return str.toLowerCase().split("").reverse().join("");
+}
+
+export function checkPalindrome(str: string) {
+    let reversed = reverseAString(str);
+
+    str = str.replace(/\s+/g, '');
+    reversed = reversed.replace(/\s+/g, '');
+    return (str == reversed) ? true : false;
```

Comment:
we can directly return the comparison expression, like this:
```typescript
return str === reversed;
```

which will be evaluated and return the evaluation result, either true or false